aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsProxyHttpServletRequest.java

```diff
@@ -471,6 +480,11 @@ public RequestDispatcher getRequestDispatcher(String s) {
 
     @Override
     public int getRemotePort() {
+        if (Objects.nonNull(request.getRequestContext().getElb())) {
+            String hostHeader = request.getHeaders().get(PORT_HEADER_NAME);
+            if (Objects.nonNull(hostHeader))
```

Comment:
please use brackets for the if statement

Reply:
Updated. Thanks